src/open_inwoner/components/templates/components/Header/NavigationAuthenticated.html

```diff
@@ -8,6 +8,11 @@
         <ul class="primary-navigation__list">
             <li class="primary-navigation__list-item">
             {% button text=_('Welkom ')|addstr:request.user.get_short_name type="button" icon="expand_more" icon_position="after" icon_outlined=True transparent=True extra_classes="primary-navigation--toggle" %}
+            {% for child in children %}
+                {% if child.indicator %}
+                    <span class="indicator"><span class="indicator__dot"></span></span>
+                {% endif %}
+            {% endfor %}
```

Comment:
This for-loop is probably not what we want, as there could be multiple menu items with indicators. 

Also there is a weird situation because the `children` context variable shouldn't be present at this point, it supposedly gets generated by `show_menu_below_id` a little further down.

I'll investigate what is going on and create a snippet of code for this.

Reply:
This seems to be a pattern. Cf. e.g. `templates/cms/menu/primary.html`: a loop over `children`, even though the context is updated by `show_menu` further down.

Reply:
It seems like a bug or flaw in django-cms where it doesn't use the context stack the way template tags are supposed to. I filled an issue https://github.com/django-cms/django-cms/issues/7628 but I don't expect anything from it.